anisette_server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -y libplist3 curl unzip wget
+
+WORKDIR /opt/
+
+RUN curl -s https://api.github.com/repos/Dadoum/Provision/releases/latest \
+ | grep "browser_download_url.*anisette_server-$(uname -m)" \
+ | cut -d : -f 2,3 \
+ | tr -d \" \
+ | wget -qi -
+RUN curl https://apps.mzstatic.com/content/android-apple-music-apk/applemusic.apk -O \
```

Comment:
Yes, so with docker you want the image to be as far build as possible, so that when you run the image, it doesn't differ (to much) from the other times you'll run it.

Reply:
Yes, but to avoid legal troubles I thought we were trying to avoid “distributing” the libraries: that’s why they don’t come with provision.
I think it still counts as distribution if it’s in a docker image.
If you download them at runtime, there is no distribution